libraries/BufferedBleSpp/README.md

```diff
@@ -0,0 +1,89 @@
+# A nanoFramefork BLE SPP server with Xamarin Forms client sample
+
+The BufferedBleSpp library is an BLE SPP implementation that allows for the transfer of large byte arrays between a Xamarin Forms client and nanoFramework device.
+
+The NordicSpp example in the nanoFramework Bluetooth BLE samples repository only allows transfers of up to 256 bytes at a time. BufferedBleSpp improves on this. It can come in handy when transferring e.g. Json, which can become quite lengthy, and one doesn't need to care too much about the size of the data. 
+
+BufferedBleSpp transfers unfortunately are slow, so should only be used under non-critical timing situations.
+
+Its been tested on a ESP32 that was able to load the ESP32_BLE_REV3 firmware.
+
+This sample also demonstrates BufferBleSpp's use in a Xamarin Forms mobile client environment.
+
+The nanoFramework project is provided as a zipped file [BufferedBleSppNanoF.zip].
+
+The Xamarin Forms project is provided as a zipped file [BufferedBleSppXam.zip].
+
+There are two objects, both called BufferedBleSpp, one for each platform:
+
+Sample Xamarin Forms code demonstrating its usage:
+```	
+messageLabel.Text = "";
+var message = "";
+for (int i = 0; i < 10000; i++) message += " " + i.ToString();
```

Comment:
even if it's a sample, please use full bracket { } and make it on a new line. Otherwise, this can be confusing. In all the .NET nanoFramework repository, we went for that rule to provide clarity.